src/Cdo.chpl

```diff
@@ -38,1601 +38,1561 @@ This module is focused on providing interfaces, base classes, and helpers that a
 
 module Cdo{
 
-use Reflection;
-use Regexp;
+    use Reflection;
+    use Regexp;
```

Comment:
Stylistic nit: Chapel programs are typically written with 2-space indentation to avoid losing all your horizontal space so quickly. We might want to switch to that for these modules in the future.

Reply:
Sure, I will try to update the codebase with the 2-space indentation.